build.mjs

```diff
@@ -19,20 +19,22 @@ const build = (options) => esbuild
 const browserConfig = {
```

Comment:
Do we need this `esbuild` build file anymore?

Reply:
I don't think we do and will remove it.